actions/getRecords.js

```diff
@@ -2,6 +2,15 @@ var crypto = require('crypto'),
     once = require('once'),
     db = require('../db')
 
+var fiveMegaBytes = 5 * 1024 * 1024
+function lessThanFiveMB(sumOfBytes) {
+  return function(item) {
+    buffer = new Buffer(item.value.Data, 'base64')
+    sumOfBytes = sumOfBytes + Buffer.byteLength(buffer.toString('ascii'))
```

Comment:
Does `sumOfBytes += buffer.length` not work? Converting the string to `ascii` seems quite specific

Reply:
Also, you can probably skip the creation and allocation of the buffer entirely here, base64 has a pretty standard length calculation: `item.value.Data.length * 0.75` (minus the number of `=` chars at the end)

Reply:
Makes sense. I feel that it is okay to ignore the trailing '=' in calculating the size of data. It will not necessary to ensure it is 100% less than 5 MB.